diff -ur squish-old/alpha.cpp squish/alpha.cpp
--- squish-old/alpha.cpp	2008-12-08 09:22:33.000000000 +0100
+++ squish/alpha.cpp	2008-12-08 09:23:07.000000000 +0100
@@ -24,6 +24,7 @@
    -------------------------------------------------------------------------- */
 
 #include "alpha.h"
+#include <limits.h>
 #include <algorithm>
 
 namespace squish {
diff -ur squish-old/colourfit.h squish/colourfit.h
--- squish-old/colourfit.h	2008-12-08 09:22:33.000000000 +0100
+++ squish/colourfit.h	2008-12-08 09:23:07.000000000 +0100
@@ -27,6 +27,7 @@
 #define SQUISH_COLOURFIT_H
 
 #include <squish.h>
+#include "limits.h"
 #include "maths.h"
 
 namespace squish {
diff -ur squish-old/config.h squish/config.h
--- squish-old/config.h	2008-12-08 09:22:33.000000000 +0100
+++ squish/config.h	2008-12-08 09:23:07.000000000 +0100
@@ -36,7 +36,7 @@
 #define SQUISH_USE_SSE 0
 #endif
 
-// Internally et SQUISH_USE_SIMD when either Altivec or SSE is available.
+// Internally set SQUISH_USE_SIMD when either Altivec or SSE is available.
 #if SQUISH_USE_ALTIVEC && SQUISH_USE_SSE
 #error "Cannot enable both Altivec and SSE!"
 #endif
